dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractWorkflowResource.java

```diff
@@ -170,6 +171,12 @@ protected void updateDBWorkflowWithSourceControlWorkflow(Workflow workflow, Work
         for (WorkflowVersion version : newWorkflow.getWorkflowVersions()) {
             // skip frozen versions
```

Comment:
This was like this, but confused by this comment -- frozen check is further below